OpenKeychain/src/main/java/org/sufficientlysecure/keychain/pgp/PgpKeyOperation.java

```diff
@@ -1192,81 +1139,38 @@ private PgpEditKeyResult internalRestricted(PGPSecretKeyRing sKR, SaveKeyringPar
 
     }
 
-
-    public PgpEditKeyResult modifyKeyRingPassphrase(CanonicalizedSecretKeyRing wsKR,
-                                                    CryptoInputParcel cryptoInput,
-                                                    ChangeUnlockParcel changeUnlockParcel) {
+    public PgpEditKeyResult addS2kToSubKeys(CanonicalizedSecretKeyRing wsKR,
+                                            Passphrase newPassphrase) {
 
         OperationLog log = new OperationLog();
         int indent = 0;
 
-        if (changeUnlockParcel.mMasterKeyId == null || changeUnlockParcel.mMasterKeyId != wsKR.getMasterKeyId()) {
-            log.add(LogType.MSG_MF_ERROR_KEYID, indent);
-            return new PgpEditKeyResult(PgpEditKeyResult.RESULT_ERROR, log, null);
-        }
-
         log.add(LogType.MSG_MF, indent,
                 KeyFormattingUtils.convertKeyIdToHex(wsKR.getMasterKeyId()));
         indent += 1;
         progress(R.string.progress_building_key, 0);
 
-        // We work on bouncycastle object level here
         PGPSecretKeyRing sKR = wsKR.getRing();
```

Comment:
`sKR` actually clashes with our naming convention for static variables (starts with `s` - [variable naming](https://source.android.com/source/code-style.html#follow-field-naming-conventions)), but I see it's what's been used everywhere else in this class, so I guess it's okay.